<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="QT-INBG-LT1958" failures="46" tests="49" name="Test" time="245.826" errors="0" timestamp="2022-10-13T14:49:48 IST">
  <testcase classname="SignIn" name="Signinemptyvalid" time="5.954">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="Session ID is null. Using WebDriver after calling quit()?
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;QT-INBG-LT1958&amp;apos;, ip: &amp;apos;10.20.15.58&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64259}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64259/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'QT-INBG-LT1958', ip: '10.20.15.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64259}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64259/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:145)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:322)
at SignIn.Signinemptyvalid(SignIn.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Signinemptyvalid -->
  <testcase classname="SignIn" name="Signinpassspecialinavalid" time="4.226">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="Session ID is null. Using WebDriver after calling quit()?
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;QT-INBG-LT1958&amp;apos;, ip: &amp;apos;10.20.15.58&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64294}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64294/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'QT-INBG-LT1958', ip: '10.20.15.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64294}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64294/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:145)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:322)
at SignIn.Signinpassspecialinavalid(SignIn.java:71)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Signinpassspecialinavalid -->
  <testcase classname="SignIn" name="Signinpominavalid" time="3.603">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="Session ID is null. Using WebDriver after calling quit()?
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;QT-INBG-LT1958&amp;apos;, ip: &amp;apos;10.20.15.58&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64329}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64329/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'QT-INBG-LT1958', ip: '10.20.15.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64329}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64329/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:145)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:322)
at SignIn.Signinpominavalid(SignIn.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Signinpominavalid -->
  <testcase classname="SignIn" name="Signinusermoretwentyinavalid" time="3.814">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="Session ID is null. Using WebDriver after calling quit()?
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;QT-INBG-LT1958&amp;apos;, ip: &amp;apos;10.20.15.58&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64365}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64365/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'QT-INBG-LT1958', ip: '10.20.15.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64365}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64365/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:145)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:322)
at SignIn.Signinusermoretwentyinavalid(SignIn.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Signinusermoretwentyinavalid -->
  <testcase classname="SignIn" name="Signinvalid" time="4.52">
    <failure type="org.openqa.selenium.NoSuchSessionException" message="Session ID is null. Using WebDriver after calling quit()?
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;QT-INBG-LT1958&amp;apos;, ip: &amp;apos;10.20.15.58&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64400}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64400/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'QT-INBG-LT1958', ip: '10.20.15.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [null, getCurrentUrl {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:64400}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:64400/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:145)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:322)
at SignIn.Signinvalid(SignIn.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- Signinvalid -->
  <testcase classname="SignIn" name="signIn" time="4.465"/>
  <testcase classname="Order" name="case0" time="4.484"/>
  <testcase classname="Order" name="case1" time="6.352">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case1(Order.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case1 -->
  <testcase classname="Order" name="case10" time="5.416">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case10(Order.java:160)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case10 -->
  <testcase classname="Order" name="case11" time="4.943">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case11(Order.java:174)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case11 -->
  <testcase classname="Order" name="case12" time="6.238">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case12(Order.java:188)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case12 -->
  <testcase classname="Order" name="case13" time="4.443">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case13(Order.java:203)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case13 -->
  <testcase classname="Order" name="case14" time="4.685">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case14(Order.java:217)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case14 -->
  <testcase classname="Order" name="case15" time="6.816">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case15(Order.java:232)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case15 -->
  <testcase classname="Order" name="case16" time="4.545">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case16(Order.java:246)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case16 -->
  <testcase classname="Order" name="case17" time="4.463">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case17(Order.java:260)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case17 -->
  <testcase classname="Order" name="case18" time="4.296">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case18(Order.java:274)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case18 -->
  <testcase classname="Order" name="case19" time="5.271">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case19(Order.java:288)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case19 -->
  <testcase classname="Order" name="case2" time="5.329">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case2(Order.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case2 -->
  <testcase classname="Order" name="case20" time="4.685">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case20(Order.java:302)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case20 -->
  <testcase classname="Order" name="case21" time="4.924">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case21(Order.java:316)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case21 -->
  <testcase classname="Order" name="case22" time="5.859">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case22(Order.java:332)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case22 -->
  <testcase classname="Order" name="case23" time="5.656">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case23(Order.java:348)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case23 -->
  <testcase classname="Order" name="case24" time="5.656">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case24(Order.java:362)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case24 -->
  <testcase classname="Order" name="case25" time="4.61">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case25(Order.java:376)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case25 -->
  <testcase classname="Order" name="case26" time="4.873">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case26(Order.java:390)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case26 -->
  <testcase classname="Order" name="case27" time="4.474">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case27(Order.java:404)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case27 -->
  <testcase classname="Order" name="case28" time="5.142">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case28(Order.java:418)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case28 -->
  <testcase classname="Order" name="case29" time="5.246">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case29(Order.java:432)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case29 -->
  <testcase classname="Order" name="case3" time="5.5">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case3(Order.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case3 -->
  <testcase classname="Order" name="case30" time="4.781">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case30(Order.java:446)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case30 -->
  <testcase classname="Order" name="case31" time="6.454">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case31(Order.java:460)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case31 -->
  <testcase classname="Order" name="case32" time="4.581">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case32(Order.java:474)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case32 -->
  <testcase classname="Order" name="case33" time="4.891">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case33(Order.java:488)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case33 -->
  <testcase classname="Order" name="case34" time="4.908">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case34(Order.java:502)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case34 -->
  <testcase classname="Order" name="case35" time="5.129">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case35(Order.java:516)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case35 -->
  <testcase classname="Order" name="case36" time="4.442">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case36(Order.java:530)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case36 -->
  <testcase classname="Order" name="case37" time="5.441">
    <failure type="java.lang.AssertionError" message="expected [ohttps://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [ohttps://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case37(Order.java:544)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case37 -->
  <testcase classname="Order" name="case4" time="5.386">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case4(Order.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case4 -->
  <testcase classname="Order" name="case5" time="5.503">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case5(Order.java:91)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case5 -->
  <testcase classname="Order" name="case6" time="4.432">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case6(Order.java:105)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case6 -->
  <testcase classname="Order" name="case7" time="5.194">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case7(Order.java:119)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case7 -->
  <testcase classname="Order" name="case8" time="5.334">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case8(Order.java:132)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case8 -->
  <testcase classname="Order" name="case9" time="6.012">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/order.html] but found [driver.getCurrentURL]
at Order.case9(Order.java:146)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- case9 -->
  <testcase classname="SignUp" name="demo" time="3.867"/>
  <testcase classname="SignUp" name="demo1" time="4.487">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/signup.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/signup.html] but found [driver.getCurrentURL]
at SignUp.demo1(SignUp.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demo1 -->
  <testcase classname="SignUp" name="demo2" time="3.791">
    <failure type="org.openqa.selenium.NoAlertPresentException" message="no such alert
  (Session info: chrome=106.0.5249.119)
Build info: version: &amp;apos;4.4.0&amp;apos;, revision: &amp;apos;e5c75ed026a&amp;apos;
System info: host: &amp;apos;QT-INBG-LT1958&amp;apos;, ip: &amp;apos;10.20.15.58&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [5398f90af15c38070fab16e3434558cc, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:49872}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:49872/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 5398f90af15c38070fab16e3434558cc">
      <![CDATA[org.openqa.selenium.NoAlertPresentException: no such alert
  (Session info: chrome=106.0.5249.119)
Build info: version: '4.4.0', revision: 'e5c75ed026a'
System info: host: 'QT-INBG-LT1958', ip: '10.20.15.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [5398f90af15c38070fab16e3434558cc, getAlertText {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 106.0.5249.119, chrome: {chromedriverVersion: 105.0.5195.52 (412c95e51883..., userDataDir: C:\Users\sruthy.sl\AppData\...}, goog:chromeOptions: {debuggerAddress: localhost:49872}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:49872/devtoo..., se:cdpVersion: 106.0.5249.119, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 5398f90af15c38070fab16e3434558cc
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:547)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:602)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:606)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.alert(RemoteWebDriver.java:1128)
at LandingPage.SignUp(LandingPage.java:96)
at SignUp.demo2(SignUp.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
  <testcase classname="SignUp" name="demo3" time="4.513">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/signup.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/signup.html] but found [driver.getCurrentURL]
at SignUp.demo3(SignUp.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demo3 -->
  <testcase classname="SignUp" name="demo4" time="6.053">
    <failure type="java.lang.AssertionError" message="expected [https://qualicoach.org/mwapp/signup.html] but found [driver.getCurrentURL]">
      <![CDATA[java.lang.AssertionError: expected [https://qualicoach.org/mwapp/signup.html] but found [driver.getCurrentURL]
at SignUp.demo4(SignUp.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demo4 -->
</testsuite> <!-- Test -->
